admin/sql/updates/2021-07-27-add-critiquebrainz-as-external-service.sql

```diff
@@ -1,3 +1 @@
-BEGIN;
 ALTER TYPE external_service_oauth_type ADD VALUE 'critiquebrainz';
-COMMIT;
```

Comment:
Not related to this PR but @alastair reported this script was failing when he tried locally hence fixing it as well here. (Can't run alter type ... add value inside transaction).